fuzzamoto/src/oracles.rs

```diff
@@ -128,3 +128,85 @@ where
         "NetSplitOracle"
     }
 }
+
+#[derive(Debug, Copy, Clone)]
+pub struct InflationOracle<TX> {
+    phantom: PhantomData<TX>,
+}
+
+impl<TX> Default for InflationOracle<TX> {
+    fn default() -> Self {
+        Self {
+            phantom: PhantomData,
+        }
+    }
+}
+
+const BTC_TO_SATOSHI: u64 = 100000000;
+
+/// compute the total number of bitcoins that has been produced at the given height
+pub fn total_coins_until(height: u64) -> f64 {
```

Comment:
I think there is an off-by-one issue in this function. Height `0` returns `0` but it should already include the coins created at height 0 (i.e. return `50`)

Reply:
I was looking at this while checking the result: https://learnmeabitcoin.com/technical/mining/block-reward/
so I think I confused it 😅